linera-execution/src/execution_state_actor.rs

```diff
@@ -233,6 +233,20 @@ where
                 let messages = self.system.open_chain(config, next_message_id)?;
                 callback.respond(messages)
             }
+
+            CloseChain {
+                application_id,
+                callback,
+            } => {
+                let mut apps = self.system.authorized_applications.get().iter();
```

Comment:
This works but we might also want to be a little more explicit & future-proof:
```
struct ApplicationAuthorizations {
     execute_operations: Option<Vec<ApplicationId>>,  // None means everyone, vec![] means nobody
     close_chain: Vec<ApplicationId>, // vec![] means nobody
}
```
(Note the nuance in the two types and their default values.)

What do you think?

Reply:
And you prefer `Vec` over `BTreeSet`? (I imagine it doesn't matter in practice, since it's never going to be a large number of entries.)

Reply:
"Permissions" sounds great. Iirc there is an issue with BTreeSet and serde-reflection but in this case it doesn't matter.